### DISCUSSION TITLE
Script attivati da pulsanti: personalizzare stato ed icona

### DISCUSSION THREAD

**Discussion by crug80** [indomus/forum#3053] [HUB: Home Assistant]
Ho creato una serie di script per aggregare in una singola azione alcune attività.. Ad esempio 1 script accende PdC ed apri elettrovalvola.. 
Ora, collegando lo script ad un pulsante nella mia dash, ovviamente sia lo stato scritto che l'icona fanno riferimento allo script. 
E' possibile in qualche modo che lo stato faccia riferimento ad una delle entità gestite (ad esempio la PdC)?

**Comment by Scialla** [comment]
Devi crearti tu una entità template collegata allo script e configurandola come ti pare per l'icona e quant'altro, oppure lo fai a livello di scheda configurando come ti pare con if e quant'altro. Quello che chiedi non mi risulta possibile

**Reply by crug80** [reply]
Non ho ben capito...  Dovrei fare un template che come stato restituisca lo stato di uno degli oggetti comandati dallo script e come azione chiamare lo script?! 
Se sì, devo studiarmi come fare.

**Comment by 50494554524F** [comment]
se vuoi fare tutto con le card di ha puoi farlo con conditional-card + 2 button-card... altrimenti qualcosa tipo https://github.com/custom-cards/button-card o simili che trovi in hacs

**Reply by crug80** [reply]
Carina la button card, la provo. Grazie
